doc/ProofGeneral.texi

```diff
@@ -4217,16 +4186,6 @@ minor mode, type @kbd{M-x abbrev-mode RET}. When you are not in Abbrev
 mode you can expand an abbreviation by pressing @kbd{C-x '}
 (@code{expand-abbrev}). See the Emacs manual for more details.
 
-Coq Proof General has a special experimental feature called "Holes"
-which makes use of the abbreviation mechanism and includes a large list
-of command abbreviations. @xref{Holes feature}, for details.  With other
-provers, you may use the standard Emacs commands above to set up your
-own abbreviation tables.
```

Comment:
Are you sure you want to remove this documentation from now on? 😉
but maybe you'd have plans to port part of the features of Holes to the same "elisp backend" as that of company-coq?

Reply:
Indeed. I made a small poll on Zulip and it appear I was the only user of this feature.
I have tested yasnippet snippets for coq and it works quite well and it does not interfere with company coc completions (even if it also uses yasnippet). There are a few editing tricks that I will miss but nevermind.

Reply:
More precisely I plan to

1. remove everything from the holes library shortly.
2. document and/or propose a set of user-customized abbreviations (unless this subsumed by company-coq maybe)
3. one day maybe try to reimplement the copy-paste-in-the-holes thing, even if nobody seems to be willing to try it...